### ISSUE TITLE
Errors showing in nvim due to hex color with 7 characters

### ISSUE THREAD

**Issue by fbontin** [nvim-tree/nvim-web-devicons#358]
As already [mentioned by two other users](https://github.com/nvim-tree/nvim-web-devicons/commit/c4a44ba8e4a562835de200a897d4e65c211e5401#r135505777), there's a color with 7 characters which is causing issues when starting neovim. I'm getting these errors directly when starting:
```
Failed to run `config` for nvim-tree.lua

...re/nvim/lazy/nvim-web-devicons/lua/nvim-web-devicons.lua:253: Invalid highlight color: '#f28ba28'

# stacktrace:
  - ~/.config/nvim/lua/user/plugins/nvim-tree.lua:10 _in_ **config**
  - init.lua:17
```

**Comment by gegoune** [comment]
Fixed in #357

**Comment by fbontin** [comment]
Nice, thanks for the extremely speedy fix 👏
